### DISCUSSION TITLE
Enable X amount of streams based on a subset of thousands of available streams?

### DISCUSSION THREAD

**Discussion by philippeboyd** [airbytehq/airbyte#33366] [Connector Questions]
Hello,

I have a database with over 1200 tables. I only want to sync (enable stream) about 200 of those tables (could maybe be 400).

I have to manually enable the little UI switches for 100s of those tables.
<img width="425" alt="image" src="https://github.com/airbytehq/airbyte/assets/3239656/017342df-5ab3-4d67-a4a6-b1633a5e8e64">

Is there a way in the javascript console to enable all streams based on a list of strings (wanted streams).

HTML code is not the prettiest and class name looks to be generated...

**Comment by pgrant87** [comment]
Hi @philippeboyd 

I'm unsure if this is possible from the javascript console but in instances like this where you want to configure a connection programmatically, I'd recommend the use of our [Airbyte Terraform provider](https://reference.airbyte.com/reference/using-the-terraform-provider)

**Comment by marcosmarxm** [comment]
There are more info about why the bulk was removed here: https://github.com/airbytehq/airbyte/discussions/32725

[a comment by an automated account is not shown]
